sdk/purview/Azure.Analytics.Purview.Share/README.md

```diff
@@ -59,15 +59,235 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 The following section shows you how to initialize and authenticate your client and get a sent share.
 
-### Get a Sent Share
+### Create sent share
 
 ```C#
 var credential = new DefaultAzureCredential();
-var client = new SentSharesClient(new Uri("https://<my-account-name>.purview.azure.com"), credential);
+var endpoint = "https://<my-account-name>.purview.azure.com";
+
+var sentShareClient = new SentSharesClient(endpoint, credential);
+
+// Create sent share
+var sentShareName = "sample-Share";
+
+var inPlaceSentShareDto = new
+{
+    shareKind = "InPlace",
+    properties = new
+    {
+        description = "demo share",
+        collection = new
+        {
+            referenceName = "<reference>",
```

Comment:
Let's add a comment.
// <my-account-name> for root collection else name of any accessible child collection in the Purview account.